Frontend/ui-driver/src/Components/BottomNavBar/Controller.purs

```diff
@@ -46,9 +45,9 @@ navData screenName = {
       activeIcon: "ny_ic_join_active,https://assets.juspay.in/nammayatri/images/driver/ic_profile_active.png",
       defaultIcon: "ny_ic_join_inactive,https://assets.juspay.in/nammayatri/images/driver/ic_profile_inactive.png",
       text: "Join"
-    },
-    {
-      activeIcon: "ic_referral_active,https://assets.juspay.in/nammayatri/images/driver/ic_referral_active.png",
+    }] else []) <> 
+    (if (getMerchant FunctionCall /= YATRISATHI) then [{
```

Comment:
create a variable in configuration and use that instead of adding a merchant check.